lib/Cake/Console/Command/Task/ExtractTask.php

```diff
@@ -151,7 +151,7 @@ protected function _getPaths() {
  */
 	public function execute() {
 		if (!empty($this->params['exclude'])) {
-			$this->_exclude = explode(',', $this->params['exclude']);
+			$this->_exclude = explode(',', str_replace('/', DS, $this->params['exclude']));
```

Comment:
No need to `array_map` because `$this->_searchFiles()` is called only when no `files` specified.